File: 2°semestre/AEDS_II/aulas/00-nivelamento/exercicios/00m-Introducao_C/Celula.c
//QUESTAO 42


#include <stdio.h>
#include <stdlib.h>
#include <string.h>

typedef struct Celula{
    int dado;
    struct Celula *prox;
}Celula;

Celula *novaCelula(int dado){
    Celula *nova = (Celula*) malloc(sizeof(Celula));
    nova->dado = dado;
    nova->prox = NULL;
    return nova;
}




int main(){



return 0;
}

File: 2°semestre/AEDS_II/testes/doidona/Celula.java
public class Celula {
    public int elemento;
    public Celula prox;

    Celula(){
        this(-1);
    }

    Celula(int elemento){
        this(elemento, null);
    }

    Celula(int elemento, Celula prox){
        this.elemento = elemento;
        this.prox = prox;
    }
}
